counter/src/pages/Functions.jsx

```diff
@@ -0,0 +1,5 @@
+export const arr=[];
```

Comment:
use created just a function .manage this in state.

Reply:
Resolved sir